backend/src/main/java/ch/puzzle/okr/util/QuarterRangeChecker.java

```diff
@@ -0,0 +1,23 @@
+package ch.puzzle.okr.util;
+
+import ch.puzzle.okr.models.Quarter;
+
+import java.time.LocalDate;
+
+public class QuarterRangeChecker {
```

Comment:
Der QuarterRangeChecker liegt im PROD-Code, wird aber nur im TEST-Code verwendet.
Evtl. in TEST-Code verschieben?

Reply:
Gute Idee. Habe ich gemacht.